cli/export.go

```diff
@@ -0,0 +1,62 @@
+package cli
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"go.keploy.io/server/v2/cli/provider"
+	"go.keploy.io/server/v2/config"
+	toolsSvc "go.keploy.io/server/v2/pkg/service/tools"
+	"go.keploy.io/server/v2/utils"
+	"go.uber.org/zap"
+)
+
+func init() {
+	Register("export", Export)
+}
+
+func Export(ctx context.Context, logger *zap.Logger, _ *config.Config, serviceFactory ServiceFactory, cmdConfigurator CmdConfigurator) *cobra.Command {
+
+	var exportCmd = &cobra.Command{
+		Use:     "export",
+		Short:   "export Keploy tests as postman collection",
+		Example: "keploy export",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			disableAnsi, _ := (cmd.Flags().GetBool("disable-ansi"))
+			provider.PrintLogo(disableAnsi)
+			return cmd.Help()
+		},
+	}
+	var postmanCmd = &cobra.Command{
+		Use:     "postman",
+		Short:   "export Keploy tests as Postman collection",
+		Example: "keploy export postman",
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			disableAnsi, _ := (cmd.Flags().GetBool("disable-ansi"))
+			provider.PrintLogo(disableAnsi)
+			svc, err := serviceFactory.GetService(ctx, "export")
+			if err != nil {
+				utils.LogError(logger, err, "failed to get service")
+				return nil
+			}
+			var tools toolsSvc.Service
+			var ok bool
+			if tools, ok = svc.(toolsSvc.Service); !ok {
```

Comment:
```suggestion
			if tools, ok := svc.(toolsSvc.Service); !ok {
```
why are we defining vars when we can do it like so.

Reply:
because of this 
<img width="749" alt="image" src="https://github.com/user-attachments/assets/160f142b-9a22-4136-9c11-858731310c3b">

would suggest running it locally and try to compile on local machine to detect breaking changes.